CAP/gap/UniversalObjects.gi

```diff
@@ -782,6 +834,74 @@ InstallMethod( Pushout,
     
 end );
 
+##
+InstallOtherMethod( PushoutProjectionFromCoproduct,
+        [ IsList ],
+                    
+  function( diagram )
+    
+    local ranges_of_diagram, cat, test_sink;
+    
+    ranges_of_diagram := List( diagram, Range );
+    
+    cat := CapCategory( diagram[1] );
```

Comment:
Please adjust this in the same way as `FiberProductEmbeddingInDirectProduct`.

Also, please remove the empty line before `local ...;` in all the newly added functions.

Reply:
Sorry, I forgot about the duals...
It's fixed now.